src/App.tsx

```diff
@@ -1,9 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter((movie) => {
+    const { description, title } = movie;
+    const lowerCaseDescription = description.toLowerCase();
+    const lowerCaseTitle = title.toLowerCase();
+    const lowerCaseQuery = query.toLowerCase().trim();
+
+    return (
+      lowerCaseDescription.includes(lowerCaseQuery)
+      || lowerCaseTitle.includes(lowerCaseQuery)
+    );
+  });
```

Comment:
looks pretty, but duplicated
DRY, let's create a dedicated function to unify functional